### ISSUE TITLE
Support Arrow Flight endpoint

### ISSUE THREAD

**Issue by Dandandan** [roapi/roapi#77]
It might make sense to have a look at supporting Arrow Flight as endpoint, this looks to be a good match with roapi.

https://arrow.apache.org/docs/format/Flight.html

**Comment by houqp** [comment]
Yes, I agree this would be a very good addition as a query frontend :)
